### PR TITLE
Revert Link to v2

### PR DESCRIPTION
**Pull request by kcasas**
/cc @zendesk/mintegrations

### Description
* revert bump v1.0.5
* revert PR #66 and PR #40

### Risks
* [low] might have missed some commits that needs to be reverted

### CODE REVIEW

File: app.css [hunk 1]
```diff
@@ -1,17 +1,32 @@
...
+.o-abandoned,.o-voided,.o-refunded,.o-restocked
+{background-color:#af9c7e;color:#fff}
+
+h5 {
+	margin-top: 9px;
 }
+
+.closed, .cancelled {
+  color: red;
+}
```
Comment (miogalang) [inline app.css]: new line!

**Comment by miogalang** [comment]
Please test on local before merging.

**Comment by samgavinio** [comment]
Do we need to delete: https://github.com/zendesk/shopify_app/releases/tag/v2.0.0

**Comment by kcasas** [comment]
@miogalang Just tested on local and it's still working.

**Pull request merged**